play-ws-standalone/src/main/java/play/libs/ws/StandaloneWSClient.java

```diff
@@ -22,6 +22,8 @@
      * properties on the StandaloneWSRequest by chaining calls, and execute the request to
      * return an asynchronous {@code CompletionStage<StandaloneWSResponse>}.
      *
+     * SHOULD NOT include the query string. Use {@link StandaloneWSRequest#addQueryParameter(String, String)} instead.
```

Comment:
I actually don't see a problem if the user passes an URL with the query string params.

Forcing them to somehow parse the URL and extract query string params does not look like a good solution either.